src/lib/Scenes/Partner/Components/__tests__/PartnerArtwork-tests.tsx

```diff
@@ -0,0 +1,43 @@
+import { PartnerArtwork_partner } from "__generated__/PartnerArtwork_partner.graphql"
+import GenericGrid from "lib/Components/ArtworkGrids/GenericGrid"
+import { renderRelayTree } from "lib/tests/renderRelayTree"
+import React from "react"
+import { graphql, RelayPaginationProp } from "react-relay"
+import { PartnerArtworkFixture } from "../__fixtures__/PartnerArtwork-fixture"
+import { PartnerArtworkFragmentContainer as PartnerArtwork } from "../PartnerArtwork"
+
+jest.unmock("react-relay")
+
+describe("PartnerArtwork", () => {
+  const getWrapper = async (partner: Omit<PartnerArtwork_partner, " $fragmentRefs">) =>
+    await renderRelayTree({
+      Component: (props: any) => {
+        return <PartnerArtwork partner={{ ...partner }} relay={{ environment: {} } as RelayPaginationProp} {...props} />
+      },
+      query: graphql`
+        query PartnerArtworkTestsQuery @raw_response_type {
+          partner(id: "anderson-fine-art-gallery-flickinger-collection") {
+            id
+            artworks: artworksConnection(first: 10) {
+              edges {
+                node {
+                  ...GenericGrid_artworks
+                }
+              }
+            }
+          }
+        }
+      `,
+      mockData: {
+        partner,
+      },
+    })
+
+  it("It renders the artworks", async () => {
```

Comment:
```suggestion
  it("renders the artworks", async () => {
```

Note that we'll need to re-do the snapshots after this change. There are also a few others we'll want to change.